.github/workflows/pr-labeler.yml

```
@@ -7,6 +7,7 @@ jobs:
     permissions:
       contents: read
       pull-requests: write
+      issues: write # permission to create labels if not exist
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)